subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -207,6 +207,7 @@ For example:
 * A task from one project directly resolves a configuration in another project.
 * A task specifies a configuration from another project as an input file collection.
 * A build script for a project resolves a configuration in another project during evaluation.
+* Project configurations are resolved from project settings
```

Comment:
what does "project settings" mean here?

Reply:
`settings.gradle(.kts)`
How would you phrase that then?